cluster/examples/kubernetes/ceph/object-test.yaml

```
@@ -20,5 +20,5 @@ spec:
   gateway:
     type: s3
     port: 80
-    securePort:
+    # securePort:
```

It's better to leave a comment that means what is the latest changes. I guess you mean the following commit.

https://github.com/rook/rook/commit/e21854145af45545b6282b5066e4018044c69382

```suggestion
    # securePort: 443
```

How about setting the default value to `securePort`. Then users can ebale securePort without setting it explicitly. In addition, could you also change the similar `securePort:` fields of the following files?

https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/object-multisite.yaml#L47
https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/object-multisite-pull-realm.yaml#L61
https://github.com/rook/rook/blob/master/Documentation/ceph-object-store-crd.md
https://github.com/rook/rook/blob/master/Documentation/ceph-object.md

---

Done.

---

Just setting the port won't be enough for a secured connection but still, it's better to give it a value.